src/dataset-builder/CohortEditor.ts

```diff
@@ -151,27 +153,37 @@ export const CriteriaView = ({ criteria, deleteCriteria, updateCriteria }: Crite
   );
 
 let criteriaCount = 1;
-const createDefaultListCriteria = (listOption: ProgramDataListOption): ProgramDataListCriteria => {
-  return {
-    kind: 'list',
-    listOption,
-    name: listOption.name,
-    id: Number(_.uniqueId('')),
-    count: 100,
-    values: [listOption.values[0]],
-  };
-};
-
-const createDefaultRangeCriteria = (rangeOption: ProgramDataRangeOption): ProgramDataRangeCriteria => {
-  return {
-    kind: 'range',
-    rangeOption,
-    name: rangeOption.name,
-    id: criteriaCount++,
-    count: 100,
-    low: rangeOption.min,
-    high: rangeOption.max,
-  };
+const createDefaultProgramDataCriteria = async (
+  datasetId: string,
+  option: SnapshotBuilderProgramDataOption
+): Promise<ProgramDataRangeCriteria | ProgramDataListCriteria> => {
+  const generatedOptions = await DatasetBuilder().getProgramDataStatistics(datasetId, option);
+  switch (generatedOptions.kind) {
+    case 'range':
+      const rangeOption = generatedOptions as ProgramDataRangeOption;
+      return {
+        kind: 'range',
+        rangeOption,
+        name: rangeOption.name,
+        id: criteriaCount++,
+        count: 100,
+        low: rangeOption.min,
+        high: rangeOption.max,
+      };
+    case 'list': {
+      const listOption = generatedOptions as ProgramDataListOption;
+      return {
+        kind: 'list',
+        listOption,
+        name: option.name,
+        id: Number(_.uniqueId('')),
+        count: 100,
+        values: [listOption.values[0]],
```

Comment:
Should we default to the first option being selected? Or default to no selection by default?

Reply:
I think no selection is better, let's go with that for now.